src/libraries/System.Private.Xml/src/System/Xml/Serialization/XmlSerializationReader.cs

```diff
@@ -15,98 +18,98 @@ namespace System.Xml.Serialization
     ///<internalonly/>
     public abstract class XmlSerializationReader : XmlSerializationGeneratedCode
     {
-        private XmlReader _r;
-        private XmlDocument _d;
-        private Hashtable _callbacks;
-        private Hashtable _types;
-        private Hashtable _typesReverse;
+        private XmlReader _r = null!;
+        private XmlDocument? _d;
+        private Hashtable _callbacks = null!;
```

Comment:
I added it for parity with other `null!` as there were so many initialized members in one of the methods that the code looked more polluted with that rather than `null!` - I felt it made it much harder to read or follow. I'll change this single instance though